.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
     steps:
       - name: Install runtime dependencies
-        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
+        run: sudo apt-get update
```

Comment:
We probably don't need this step at all. `update` only refreshes the package lists.